plugin/pkg/scheduler/scheduler.go

```diff
@@ -222,7 +225,7 @@ func (sched *Scheduler) bind(assumed *v1.Pod, b *v1.Binding) error {
 	if err != nil {
```

Comment:
Also happy to break that commit out as a separate PR since it's an independent problem & this is getting kind of big, not sure which is better